src/comgr.cpp

```diff
@@ -38,7 +38,7 @@
 #include <miopen/solver/implicitgemm_util.hpp>
 #include <miopen/stringutils.hpp>
 
-#include <amd_comgr.h>
+#include <amd_comgr/amd_comgr.h>
```

Comment:
```suggestion
#if HIP_PACKAGE_VERSION_FLAT >= 4004000000ULL
#include <amd_comgr/amd_comgr.h>
#else
#include <amd_comgr.h>
#endif
```

Reply:
The new location will be used in releases starting from 5.4.0, and in mainline starting from 5.5.